conf/application.conf

```diff
@@ -2,10 +2,10 @@ play.application.name="atom-workshop"
 
 play.application.loader=AppLoader
 
-play.http.secret.key="changeme"
+play.http.secret.key="Value must be >9 chars. If the PLAY_SESSION cookie is *used*, use https://github.com/guardian/play-secret-rotation"
 
 panda.system="atom-workshop"
 
-parsers.text.maxLength=200kB
+play.http.parser.maxMemoryBuffer=200K
```

Comment:
Prompted by this error message in the logs:

> application.conf @ file:/usr/share/atom-workshop/conf/application.conf: 9: parsers.text.maxLength is deprecated, use play.http.parser.maxMemoryBuffer instead

Apparently the `parsers.text.maxLength property` has been deprecated since Play 2.4 and https://github.com/playframework/playframework/pull/4093 - [`play.http.parser.maxMemoryBuffer` is preferred](https://www.playframework.com/documentation/2.9.x/ScalaBodyParsers#Max-content-length).

In atom-workshop, the increased size was set to allow bigger chart atoms: https://github.com/guardian/atom-workshop/pull/272